src/components/tests/ComboChart.spec.tsx

```diff
@@ -19,4 +20,49 @@ describe("ComboChart", () => {
         );
         expect(wrapper.find(AfmComboChart)).toHaveLength(1);
     });
+
+    it("should render with columnMeasures & lineMeasures", () => {
+        const wrapper = shallow(<ComboChart projectId="foo" columnMeasures={[M3]} lineMeasures={[M4]} />);
+        expect(wrapper.find(AfmComboChart)).toHaveLength(1);
```

Comment:
I would expect, that this will test 'props' of AfmComboChart, not only its presence in react tree